README.md

```diff
@@ -141,9 +141,13 @@ val records = sc.newAPIHadoopFile(
 
 ### AWS Credentials
 
-`spark-redshift` reads and writes data to S3 when transferring data to/from Redshift. As a result, it requires AWS credentials with read and write access to a S3 bucket (specified as `tempdir` in the configuration parameters described below).
+`spark-redshift` reads and writes data to S3 when transferring data to/from Redshift. As a result, it requires AWS credentials with read and write access to a S3 bucket (specified using the `tempdir` configuration parameter). Assuming that Spark has been configured to access S3, `spark-redshift` should automatically discover the proper credentials to pass to Redshift.
 
-You can provide AWS credentials via the parameters listed below, with Hadoop `fs.*` configuration settings, or by making them available via the usual environment variables, system properties or IAM roles.
+There are three ways of configuring AWS credentials for use by this library:
+
+1. Specify AWS keys via [Hadoop configuration properties](https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md). For example, if your `tempdir` configuration points to a `s3n://` filesystem then you can set the `fs.s3n.awsAccessKeyId` and `fs.s3n.awsSecretAccessKey` properties in a Hadoop XML configuration file or call `sc.hadoopConfig.set()` to mutate Spark's global Hadoop configuration.
```

Comment:
Out of user empathy, I should probably add some copy-paste-able examples of mutating `sc.hadoopConf` in Scala for both the `s3n` and `s3a` filesystems.

Reply:
Should there be a line added here to the readme about avoiding logging the aws credentials in the UNLOAD ... WITH CREDENTIALS statement of the aws credentials when authenticating by this method